.github/policies/resourceManagement.yml

```diff
@@ -205,7 +205,47 @@ configuration:
       - addReply:
           reply: This issue has been marked as "Waiting on Author" and has not had any activity for **7 day**. It has been closed for housekeeping purposes.
       - closeIssue
+    - description: Resolution no activity 
+      frequencies:
+      - hourly:
+          hour: 12
+      filters:
+      - isOpen
+      - isIssue
+      - hasLabel:
+          label: Resolution-No Activity
+      - noActivitySince:
+          days: 7
+      actions:
+      - addReply:
+          reply: This issue has been marked as "No Activity". It has been closed for housekeeping purposes.
+      - closeIssue
+    - description: This is for closing issues that have no activity for 6 months.
+      frequencies:
+      - hourly:
+          hour: 24
+      filters:
+      - isIssue
+      - isOpen
+      - noActivitySince:
+          days: 180
+      actions:
+      - addReply:
+          reply: This issue has not had any activity in 6 months, if this is a bug please try to reproduce on the latest version of PowerShell and reopen a new issue and reference this issue if this is still a blocker for you. 
+      - addLabel:
+            label: Resolution-No Activity
     eventResponderTasks:
+    - if:
+      - payloadType: Issue_Comment
+      - isActivitySender:
+          issueAuthor: True
+      - hasLabel:
+          label: Resolution-No Activity
+      then:
+      - removeLabel:
+          label: Resolution-No Activity
+      - reopenIssue

```

Comment:
this step should probably not occur if the issue is already opened.  Is there a way to check that status is closed before this step? maybe a filter, `isClosed` (if that filter exists)

Reply:
I don't think we should reopen for a comment.  Maybe add a comment that suggested the user reopen, or open a new issue.

The comment may just be mitigations.  We don't know.

Reply:
This logic takes off the resolution tag during the 7 day grace period after there is 6 months of no activity. If the tag is on an issue, it will be closed after 7 days of no activity.